cpp/scripts/run-clang-format.py

```diff
@@ -69,28 +95,32 @@ def runClangFormat(src, dst, exe):
         try:
             subprocess.check_call(cmd, shell=True)
         except subprocess.CalledProcessError:
-            print("Unable to run clang-format!"
-                  " Please configure your environment.")
+            print("Unable to run clang-format! Maybe your env is not "
+                  "configured properly?")
             raise
     # run the diff to check if there are any formatting issues
     cmd = "diff -q %s %s >/dev/null" % (src, dst)
     try:
         subprocess.check_call(cmd, shell=True)
     except subprocess.CalledProcessError:
-        print("clang-format failed!"
-              " Run 'diff %s %s' to see the formatting issues!" %
+        print("clang-format failed! Run 'diff %s %s' to know more about the "
```

Comment:
Should we just print the actual message here automatically? Seems like it would be good to know.

Reply:
That'd be a bunch of 'diff' outputs, in this case. Not sure how those would be helpful here.